frontend/__tests__/routes/_public+/date-of-birth.test.tsx

```diff
@@ -119,6 +124,8 @@ describe('_public.apply.id.date-of-birth', () => {
       vi.mocked(isPast).mockReturnValueOnce(true);
       vi.mocked(parse).mockReturnValueOnce(new Date(2000, 0, 1));
       vi.mocked(differenceInYears).mockReturnValueOnce(64).mockReturnValueOnce(64);
+      vi.mocked(parseDateString).mockReturnValue({ year: '2000', month: '01', day: '01' });
+      vi.mocked(getAgeFromDateString).mockReturnValueOnce(new Date().getFullYear() - 2000);
```

Comment:
This shouldn't return a value based on `new Date()`. It should return a consistent value across all test runs.

Reply:
Fixed